components/automate-gateway/handler/iam/v2/introspect/introspect_test.go

```diff
@@ -257,5 +256,5 @@ func testServerAndHandler(t *testing.T) (
 		s,
 		introspect.NewServer(
 			nil,
-			authorizer.NewAuthorizer(authv2.AuthorizationHandler(client)))
+			authv2.AuthorizationHandler(client))
```

Comment:
we have a card down the road to "neutralize" the v2 stuff (there's no v1, so . . .). I'm not sure it really matters and we could sweep this later, but would be cool to name this import `auth` as a start, so there's no immediate code-reader impulse to differentiate